Makefile

```diff
@@ -53,6 +54,9 @@ $(GENERATED_FILES): %: %.in Makefile
 		-e 's|[@]bindir[@]|$(BINDIR)|g' \
 		-e 's|[@]ccagent[@]|$(TARGET)|g' \
 		"$<" > "$@"
+check:
+	$(MKDIR)/.ci/go-test.sh
+	$(MKDIR)/.ci/go-lint.sh
```

Comment:
We want the linter test first because we don't want to test the code if the syntax is wrong.